src/modules/access-wallet/software/components/AccessWalletMnemonic.vue

```diff
@@ -120,11 +120,6 @@
             <div class="headline font-weight-bold">
               Select HD Derivation Path
             </div>
```

Comment:
can you add "mb-5" class to Select HD Derivation Path div (otherwise there is no space once you removed bottom div